rust/blockstore/src/arrow/block/delta/delta.rs

```diff
@@ -24,21 +27,35 @@ pub struct BlockDelta {
     pub(in crate::arrow) id: Uuid,
```

Comment:
maybe there's a way to describe the constraint that this must be backed by a BTreeMap?

Reply:
rename BlockDelta -> UnorderedDelta